consensus/src/experimental/mod.rs

```diff
@@ -20,6 +20,7 @@
 //                               │ Decision    │ (Asynchronously)
 //                               └─────────────┘
 #![allow(dead_code)]
+mod buffer_manager;
```

Comment:
maybe update the diagram above (or later)

Reply:
added to todo list